main.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+
+from app.modbus_client import ModbusClient
+from app.mqtt_client import MqttClient
+from app.configuration import Configuration
+
+SLAVE=0x01
+
+BROKER_HOST = os.getenv('BROKER_HOST')
+BROKER_PORT = int(os.getenv('BROKER_PORT'))
+MODBUS_HOST = os.getenv('MODBUS_HOST')
+MODBUS_PORT = int(os.getenv('MODBUS_PORT'))
+COMMANDS_TOPIC = os.getenv('COMMANDS_TOPIC')
+CONFIGURATION_PATH = os.getenv('CONFIGURATION_PATH', "config/example_configuration.yaml")
```

Comment:
let's have a default value, but allow to pass by the configuration path as a CLI ARG value.
```
./main.py --config-path /some/path
```

Reply:
we can also do that in a separate PR though